packages/react/src/index.js

```diff
@@ -58,8 +58,10 @@ const Card = props => {
     [lazy]
   )
   const lazyOptions = useMemo(() => (isObject(lazy) ? lazy : undefined), [lazy])
-  const [hasIntersected, cardRef] = useIntersectionObserver(
+  const cardRef = React.useRef(null)
```

Comment:
**suggestion:** I know it's recommended somewhere in the docs to import the specific module, i.e. `useRef`, rather than taking it from the default `React` import.